packages/paste-website/src/component-examples/SkeletonLoaderExamples.ts

```diff
@@ -0,0 +1,338 @@
+  const [loaded, setLoaded] = React.useState(false);
+
+  return (
+    <>
+      {loaded ? <CalendarIcon decorative={false} title="Calendar icon" /> : <SkeletonLoader size="sizeIcon30" />}
+      <Box marginTop="space60">
+        <Stack orientation="horizontal" spacing="space60">
+          <Button variant="secondary" size="small" onClick={() => setLoaded(true)}>
+            Load content
+          </Button>
+          <Button variant="secondary" size="small" onClick={() => setLoaded(false)}>
+            Reset
+          </Button>
+        </Stack>
+      </Box>
+    </>
+  );
+};
+
+render(
+  <SkeletonLoaderIcon />
+)
+`.trim();
+
+export const CardExample = `
+const SkeletonLoaderCard = () => {
+  const [loaded, setLoaded] = React.useState(false);
+
+  return (
+    <>
+      {loaded ? (
+        <Box width="500px">
+          <Card>
+            <Heading as="h1" variant="heading10">
+              Impossible
+            </Heading>
+            <Paragraph>Impossible is just a big word thrown around by small men who find it easier to live in the world they’ve been given than to explore the power they have to change it. Impossible is not a fact. It’s an opinion. Impossible is not a declaration. It's a dare. Impossible is potential. Impossible is temporary. Impossible is nothing.</Paragraph>
+          </Card>
+        </Box>
+      ) : (
+        <Box width="500px">
+          <Card>
+            <SkeletonLoader height="44px" width="100px" marginBottom="space80" />
+            <SkeletonLoader marginBottom="space20" />
+            <SkeletonLoader marginBottom="space20" />
+            <SkeletonLoader marginBottom="space20" />
+            <SkeletonLoader marginBottom="space20" />
+            <SkeletonLoader width="200px" marginBottom="space80" />
+          </Card>
+        </Box>
+      )}
+      <Box marginTop="space60">
+        <Stack orientation="horizontal" spacing="space60">
+          <Button variant="secondary" size="small" onClick={() => setLoaded(true)}>
+            Load content
+          </Button>
+          <Button variant="secondary" size="small" onClick={() => setLoaded(false)}>
+            Reset
+          </Button>
+        </Stack>
+      </Box>
+    </>
+  );
+};
+
+render(
+  <SkeletonLoaderCard />
+)
+`.trim();
+
+export const TableExample = `
+const SkeletonLoaderTable = () => {
+  const [loaded, setLoaded] = React.useState(false);
+
+  return (
+    <>
+      <Table>
+        <THead>
+          <Tr>
+            <Th width="300px">Contacts</Th>
+            <Th>Date Added</Th>
+            <Th>Last Opened</Th>
+          </Tr>
+        </THead>
+        <TBody>
+          <Tr>
+            <Td>{loaded ? <Text as="span">Adriana Lovelock</Text> : <SkeletonLoader width="113px" />}</Td>
+            <Td>{loaded ? <Text as="span">2020-09-17</Text> : <SkeletonLoader width="86px" />}</Td>
+            <Td>{loaded ? <Text as="span">2020-06-28</Text> : <SkeletonLoader width="86px" />}</Td>
```

Comment:
Setting perfect px values is a little unrealistic. Maybe we can set these to randomized (between like 60-100) % values instead?

Reply:
Are you saying have the width randomly set in the component, or just in this example use percentages instead of px values?

Reply:
Just in this example